sat/apiclient/telemetry.py

```diff
@@ -26,7 +26,7 @@
 """
 import logging
 
-from sat.apiclient.gateway import APIError, APIGatewayClient, ReadTimeout
+from csm_api_client.service.gateway import APIError, APIGatewayClient, ReadTimeout
```

Comment:
PyCharm doesn't like the signature of the `TelemetryAPIClient.stream` method, saying "Signature of method 'TelemetryAPIClient.stream()' does not match signature of the base method in class 'APIGatewayClient'". However, this same warning exists prior to your change to use the common library, so perhaps not an issue.

Reply:
Yeah, this is true. For reference, the `APIGatewayClient.stream()` method has the following type:
```python
def stream(self, *args: str, params: Optional[Dict] = None, **kwargs: Any) -> Response: ...
```
The `TelemetryAPIClient.stream()` method has the following incompatible type:
```python
def stream(self, topic: str, timeout: int, params: Optional[Dict] = None) -> Response: ...
```
As you note, this was the case before the refactor as well, though I don't think it has a practical effect in how we use the telemetry client class. (It _does_ violate the [Liskov substitution principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle) though, so strictly speaking it's not ideal.)